### PR TITLE
Add systemd support for late revisions of ubuntu.

### PR DESCRIPTION
**Pull request by elijah**
This change will make the sonarr cookbook work on e.g. a wily64 image.

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -15,9 +15,24 @@
...
-template '/etc/init/sonarr.conf' do
-  source 'init.conf.erb'
-  variables user: node[:sonarr][:user]
+case node['platform']
+when 'ubuntu'
+  if node['platform_version'].to_f >= 15.04
+	template '/etc/systemd/system/sonarr.service' do
+	  source 'sonarr.service.erb'
+	  variables user: node[:sonarr][:user]
+	end
```
Comment (jeroenj) [inline recipes/default.rb]: Could you change indenting to 4 spacers here (indenting two from the `if` above)?

File: recipes/default.rb [hunk 2]
```diff
@@ -15,9 +15,24 @@
...
+  elsif node['platform_version'].to_f >= 12.04
+	template '/etc/init/sonarr.conf' do
+	  source 'init.conf.erb'
+	  variables user: node[:sonarr][:user]
+	end
```
Comment (jeroenj) [inline recipes/default.rb]: Could you also change indenting to 4 spacers here (indenting two from the `if` above)?

**Comment by jeroenj** [comment]
Looks good. I'll merge this and release a new version after you change the indenting. :+1:

**Comment by elijah** [comment]
Done - thanks for catching.

Sorry for the flurry of seemingly-small requests -- as I've been pulling
some of these cookbooks into use, I've just been sending patches as they
come up.  I know there are a whole bunch of things yet to do -- I'm
traveling next week so I may have some time in my hotel at night to work on
things.  :)

best,

--e

**Comment by jeroenj** [comment]
No worries. I'm very happy to receive patches. Even if they're this small. Currently the cookbooks do just enough for my needs but they certainly could use some improvements. I'm very happy with any contribution made since I only have limited free time for this kind of stuff. :)

**Comment by jeroenj** [comment]
Could you squash your commits? I'll merge it afterwards. :)

**Comment by elijah** [comment]
Squashed.

**Comment by jeroenj** [comment]
Thanks. Released v1.0.2. :)

**Pull request merged**